core/src/main/java/com/amplifyframework/logging/LoggingCategory.java

```diff
@@ -30,6 +34,10 @@
  */
 public final class LoggingCategory extends Category<LoggingPlugin<?>> implements LoggingCategoryBehavior {
     private final LoggingPlugin<?> defaultPlugin;
+    // List of BroadcastLoggers created.
+    private final List<BroadcastLogger> loggers;
+    // Indicates whether the logs from all loggers should be stored.
+    private boolean storeLogs;
```

Comment:
If you have a Persistent logging plugin included, then this should be implicitly true, based on the presence of the plugin. In other words, you wouldn't need this state boolean.